test/steps/kogitoapp.go

```diff
@@ -65,7 +63,7 @@ func (data *Data) createService(runtimeType, serviceName string) error {
 }
 
 func (data *Data) createServiceWithConfiguration(runtimeType, serviceName string, table *messages.PickleStepArgument_PickleTable) error {
-	kogitoApp := framework.GetKogitoAppStub(data.Namespace, serviceName)
+	kogitoApp := framework.GetKogitoAppStub(data.Namespace, runtimeType, serviceName)
 	if err := configureKogitoAppFromTable(table, kogitoApp); err != nil {
 		return err
 	}
```

Comment:
Please remove `kogitoApp.Spec.Runtime = v1alpha1.RuntimeType(runtimeType)` on line 71 as it is not needed any more.

Reply:
ops, sorry. Removed.